feat: bring publishingv3 to feature parity

kevinslin opened the pull request:
See the demo [here](https://wiki-next.dendron.so/)  

- Handle navigating to home page in url
  - Support nav_order prop
  - Handle navigating to home page in tree view 
  - Make Lookup change value when navigating 
  - Fixed Header  #
  - Basic seo options
  - Edit on Github Link
  - Clicking on lookup should show results at level
  - Favicon 
  - Hide breadcrumb on home page
  - Planted by dendron footer
  - support static assets
- [[support logo|scratch.2021.08.14.123031.support-logo]]
- show spinner only after timeout
- show backlinks
- show children
- support canonical base url

namjul reviewed (commented):
- flashy content https://www.dropbox.com/s/xvz2xozlpoq1hn0/nextjs-template.ui.flashing-content.gif?dl=0
- scroll position in sidebar on page change https://www.dropbox.com/s/ouvlmad4vio9nig/nextjs-template.ui.scroll-position.gif?dl=0
- a way to get to the homepage: click on logo to homepage
- demo subpages are not accessible https://wiki-next.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1.html
- had some issues with the treeview in die sidebar with my own workspace
  - click on a parent did not open it
  - opening a subpage from a hierarchy (with lookup) openened the parent but could also not close/open it with the mouse.
  - I wanted to capture a gif but could not export my workspace again because [this](https://www.dropbox.com/s/y5pf03oyckogmcz/error.nextjs-template-pod-export.png?dl=0) error happens.
    - no idea what changed because it just worked before.

kevinslin commented:
> flashy content https://www.dropbox.com/s/xvz2xozlpoq1hn0/nextjs-template.ui.flashing-content.gif?dl=0

This should be fixed now.

> scroll position in sidebar on page change https://www.dropbox.com/s/ouvlmad4vio9nig/nextjs-template.ui.scroll-position.gif?dl=0

ditto

> a way to get to the homepage: click on logo to homepage
fixed

> demo subpages are not accessible https://wiki-next.dendron.so/notes/678c77d9-ef2c-4537-97b5-64556d6337f1.html
fixed

> had some issues with the treeview in die sidebar with my own workspace
    click on a parent did not open it
    opening a subpage from a hierarchy (with lookup) openened the parent but could also not close/open it with the mouse.
    I wanted to capture a gif but could not export my workspace again because this error happens.
        no idea what changed because it just worked before.

hmm, made some updates. let me know if you can still repro

namjul commented:
- :green_circle: flashy content and scroll position works now.
- :green_circle: click on homepage works
- currently on my machine I cannot export the nextjs-template pod. It now results into this error: 
![image](https://user-images.githubusercontent.com/328836/129719791-4ce913df-d58d-482b-8a6a-16c809b903eb.png)
I looks like the require note props check fails with the field `anchors` being not defined.
Should I create a bug-issue for that?
